core/import_data/import_records_section_C.py

```diff
@@ -0,0 +1,196 @@
+    {
+        "previous": "2021 Previous year price",
+        "year": 2021,
+        "remarks": "2021 Remarks",
+    },
+    {
+        "previous": "2022 Previous year price",
+        "year": 2022,
+        "remarks": "2022 Remarks",
+    },
+]
+
+
+logger = logging.getLogger(__name__)
+FILE_NAME = "SectionC.xlsx"
+
+
+def check_headers(df):
+    for c in REQUIRED_COLUMNS:
+        if c not in df.columns:
+            logger.error("Invalid column list.")
+            logger.warning(f"The following columns are required: {REQUIRED_COLUMNS}")
+            return False
+    return True
+
+
+# pylint: disable=R1702
+def parse_sheet(df):
+    """
+    parse the sheet and import the data in database
+    @param df = pandas dataframe
+    @param file_details = dict (file_name, session, convert_to_mt)
+    """
+    if not check_headers(df):
+        logger.error("Couldn't parse this sheet")
+        return
+
+    current_country = {
+        "name": None,
+        "obj": None,
+    }
+    for index_row, row in df.iterrows():
+        # another country => another country program
+        if row["Country"] != current_country["name"]:
+            current_country["name"] = row["Country"]
+            current_country["obj"] = get_country(
+                current_country["name"], index_row, logger
+            )
+
+        # we didn't found a country in db:
+        if not current_country["obj"]:
+            continue
+
+        chemical_name = row["Controlled Substances"]
+
+        # I dont know what to do with this.
+        if chemical_name == "HFC-365mfc (93%)/HFC-227ea (7%) - mezcla":
+            continue
+
+        substance, blend = get_chemical(chemical_name, index_row, logger)
+        if not substance and not blend:
+            continue
+
+        previous_year_prices_obj = None
+        for report_details in REPORT_COLUMNS:
+            cp_report = get_cp_report(
+                report_details["year"],
+                current_country["obj"].name,
+                current_country["obj"].id,
+            )
+
+            try:
+                previous_year_price = (
+                    float(row[report_details["previous"]])
+                    if row[report_details["previous"]]
+                    else None
+                )
+                # some years are digits and some are strings
+                current_year = report_details["year"]
+                try:
+                    current_year_text = row[current_year]
+                    current_year_price = (
+                        float(current_year_text) if current_year_text else None
+                    )
+                except KeyError:
+                    current_year_text = row[str(current_year)]
+                    current_year_price = (
+                        float(current_year_text) if current_year_text else None
+                    )
+            # some price values are not decimals. skip them for now
+            except ValueError:
+                logger.warning(
```

Comment:
hmmm daca previous_year are un string in loc de float, datele anului curent nu vor mai fi completate.
Daca in cazul asta vrem doar sa afisam un warning si sa nu inseram nimic atunci ca va trebui sa adaugi un continue dupa ce loghezi altfel va trebui rescrisa partea asta pentru ca datele nu vor fi bine importate